libs/methodologies/bold/recycling/organic/types/src/document-event.types.ts

```
@@ -1,64 +1,35 @@
-import { type UnknownObject } from '@carrot-fndn/shared/types';
-import { tags } from 'typia';
+import type {
+  MethodologyDocumentEvent as MethodologyDocumentEvent,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)